algorithms/sort/bitonic_sort.py

```diff
@@ -0,0 +1,40 @@
+def bitonic_sort(arr, reverse=False):
+    """
```

Comment:
Thank you for pointing out my code's weakness
Do you mean that if array size is not power of 2, then return -1 or else because it returns wrong answer?

Reply:
returning -1 would make it implicit.
but we want explicit error.
like so:
```python
# checks if n is power of two
if (n and (not(n & (n - 1))) ):
    raise ValueError("the size of input should be power of two")
```
you can just copy and paste the code above.

Reply:
my bad, it has to be `not (n and (not(n & (n - 1))) )` not `(n and (not(n & (n - 1))) )`